app/models/pacbio/run.rb

```diff
@@ -99,10 +99,19 @@ def update_smrt_link_options(options)
 
     # Returns all the used aliquots
     # @return [Array<Aliquot>] an array of the used aliquots in a run
-    def used_aliquots
+    def aliquots_to_publish_on_run
+      to_publish = []
       plates.flat_map do |plate|
-        plate.wells.flat_map(&:used_aliquots)
+        source_aliquots = plate.wells.flat_map(&:used_aliquots).select do |aliquot|
```

Comment:
Is it possible to give more intuitive name for source_aliquots - used_aliquots_with_pool_source?